openscoring-common/pom.xml

```diff
@@ -5,11 +5,12 @@
 	<parent>
 		<groupId>org.openscoring</groupId>
 		<artifactId>openscoring</artifactId>
-		<version>1.3-WND-1-SNAPSHOT</version>
+		<version>1.3-WND-2-SNAPSHOT</version>
 	</parent>
 
 	<groupId>org.openscoring</groupId>
 	<artifactId>openscoring-common</artifactId>
+	<version>1.3-WND-2-SNAPSHOT</version>
```

Comment:
If you update the dependencyManagement element (as mentioned below) you don't need to do it here